Multisig/Data/Services/Utils/HTTPClient.swift

```diff
@@ -125,7 +125,7 @@ class HTTPClient {
     }
 
     private func send(_ request: URLRequest) -> URLDataTaskResult {
-        dispatchPrecondition(condition: .onQueue(.global()))
+        dispatchPrecondition(condition: .notOnQueue(.main))
```

Comment:
isn't it the same?

Reply:
no, this breaks when you create a custom queue

Reply:
👍